.circleci/config.yml

```diff
@@ -473,9 +473,9 @@ jobs:
                 command: make libp2p_helper
             - run:
                   name: Running test -- fake_hash:full-test
-                  command: ./scripts/skip_if_only_frontend_or_rfcs.sh bash -c 'source ~/.profile && ./scripts/test.py run "fake_hash:full-test"'
+                  command: ./scripts/skip_if_only_frontend_or_rfcs.sh bash -c 'source ~/.profile && ./scripts/test.py run --non-interactive --collect-artifacts "fake_hash:full-test"'
```

Comment:
I wonder if `--non-interactive` should be the default, to simplify this file (and prevent errors in case the flag is omitted).

Reply:
But interactive is preferred when running locally. If there ever was a missing case of `--non-interactive` and it caused an issue, it wouldn't pass CI, so I don't see it as an issue so long as it can't get introduced into develop.